pkgs/racket-doc/scribblings/reference/equality.scrbl

```diff
@@ -531,6 +531,27 @@ and enables some cycle detection.
 ]
 ]
 
+Don't use the third argument to ``recur'' on counts of
+elements.
+When a data structure cares about discrete numbers, it can
+use @racket[=] on those, not @racket[equal?] or ``recur''.
```

Comment:
This doesn't explain _why_ this is what you should do. The prose needs to motivate this advice by explaining how not following it can cause problems with `check-within`.

Reply:
I added this sentence after:

> Using "recur" on counts is bad when a "recur" argument from `equal?/recur` is too tolerant on numbers within some range of each other.